console/react/src/overview/dashboard/throughputChart.test.js

```diff
@@ -33,13 +36,9 @@ it("renders the ThroughputChart component", () => {
   login(() => {
     expect(service.management.connection.is_connected()).toBe(true);
 
-    const { getByLabelText, queryByLabelText } = render(<ThroughputChart {...props} />);
-
-    // the component should initially render
-    // blank until it gets the contianer's width.
-    expect(queryByLabelText("throughput-chart")).toBe(null);
```

Comment:
The `ariaLabel` does not seem to be doing anything, neither in Jest, nor in Google Chrome.

This is what I see rendered in Chrome

```
<div class="chart-container"><div class="dashboard-chart"><div class="pf-c-chart" style="height: 100%; width: 100%; pointer-events: none; touch-action: none; position: relative;"><svg width="659" height="100" role="img" aria-labelledby="victory-container-1-title" aria-describedby="victory-container-1-desc" viewBox="0 0 659 100" style="pointer-events: all; width: 100%; height: auto;"><title id="victory-container-1-title">Messages delivered per second</title><desc id="victory-container-1-desc">overview chart</desc><g clip-path="url(#victory-clip-2)" style="height: 100%; width: 100%;"><defs><clipPath id="victory-clip-2"><rect vector-effect="non-scaling-stroke" x="0" y="0" width="659" height="100"></rect></clipPath></defs><g><path d="M0,100L11.169491525423728,100L22.338983050847457,100L33.50847457627119,100L44.67796610169491,100L55.847457627118644,100L67.01694915254238,100L78.1864406779661,100L89.35593220338983,100L100.52542372881356,100L111.69491525423729,100L122.86440677966102,100L134.03389830508476,100L145.20338983050848,100L156.3728813559322,100L167.54237288135593,100L178.71186440677965,100L189.8813559322034,100L201.05084745762713,100L212.22033898305082,100L223.38983050847457,100L234.5593220338983,100L245.72881355932205,100L256.8983050847458,100L268.0677966101695,100L279.2372881355932,100L290.40677966101697,100L301.5762711864407,100L312.7457627118644,100L323.91525423728814,100L335.08474576271186,100L346.25423728813564,100L357.4237288135593,100L368.59322033898303,100L379.7627118644068,100L390.9322033898305,100L402.10169491525426,100L413.271186440678,100L424.44067796610165,100L435.6101694915254,100L446.77966101694915,100L457.94915254237293,0L469.1186440677966,25L480.2881355932203,25L491.4576271186441,50L502.62711864406776,25L513.7966101694916,25L524.9661016949152,25L536.135593220339,50L547.3050847457628,50L558.4745762711864,25L569.6440677966102,25L580.8135593220339,25L591.9830508474577,50L603.1525423728814,50L614.3220338983051,25L625.4915254237288,25L636.6610169491526,25L647.8305084745763,50L659,50L659,100L647.8305084745763,100L636.6610169491526,100L625.4915254237288,100L614.3220338983051,100L603.1525423728814,100L591.9830508474577,100L580.8135593220339,100L569.6440677966102,100L558.4745762711864,100L547.3050847457628,100L536.135593220339,100L524.9661016949152,100L513.7966101694916,100L502.62711864406776,100L491.4576271186441,100L480.2881355932203,100L469.1186440677966,100L457.94915254237293,100L446.77966101694915,100L435.6101694915254,100L424.44067796610165,100L413.271186440678,100L402.10169491525426,100L390.9322033898305,100L379.7627118644068,100L368.59322033898303,100L357.4237288135593,100L346.25423728813564,100L335.08474576271186,100L323.91525423728814,100L312.7457627118644,100L301.5762711864407,100L290.40677966101697,100L279.2372881355932,100L268.0677966101695,100L256.8983050847458,100L245.72881355932205,100L234.5593220338983,100L223.38983050847457,100L212.22033898305082,100L201.05084745762713,100L189.8813559322034,100L178.71186440677965,100L167.54237288135593,100L156.3728813559322,100L145.20338983050848,100L134.03389830508476,100L122.86440677966102,100L111.69491525423729,100L100.52542372881356,100L89.35593220338983,100L78.1864406779661,100L67.01694915254238,100L55.847457627118644,100L44.67796610169491,100L33.50847457627119,100L22.338983050847457,100L11.169491525423728,100L0,100Z" role="presentation" shape-rendering="auto" style="fill: rgb(153, 194, 235); fill-opacity: 1; stroke: none; width: 5px; stroke-width: 2;"></path><path d="M0,100L11.169491525423728,100L22.338983050847457,100L33.50847457627119,100L44.67796610169491,100L55.847457627118644,100L67.01694915254238,100L78.1864406779661,100L89.35593220338983,100L100.52542372881356,100L111.69491525423729,100L122.86440677966102,100L134.03389830508476,100L145.20338983050848,100L156.3728813559322,100L167.54237288135593,100L178.71186440677965,100L189.8813559322034,100L201.05084745762713,100L212.22033898305082,100L223.38983050847457,100L234.5593220338983,100L245.72881355932205,100L256.8983050847458,100L268.0677966101695,100L279.2372881355932,100L290.40677966101697,100L301.5762711864407,100L312.7457627118644,100L323.91525423728814,100L335.08474576271186,100L346.25423728813564,100L357.4237288135593,100L368.59322033898303,100L379.7627118644068,100L390.9322033898305,100L402.10169491525426,100L413.271186440678,100L424.44067796610165,100L435.6101694915254,100L446.77966101694915,100L457.94915254237293,0L469.1186440677966,25L480.2881355932203,25L491.4576271186441,50L502.62711864406776,25L513.7966101694916,25L524.9661016949152,25L536.135593220339,50L547.3050847457628,50L558.4745762711864,25L569.6440677966102,25L580.8135593220339,25L591.9830508474577,50L603.1525423728814,50L614.3220338983051,25L625.4915254237288,25L636.6610169491526,25L647.8305084745763,50L659,50" role="presentation" shape-rendering="auto" style="fill: none; fill-opacity: 1; stroke: rgb(74, 95, 115); width: 5px; stroke-width: 2;"></path></g></g></svg><div style="z-index: 99; position: absolute; top: 0px; left: 0px; width: 100%; height: auto;"><svg width="659" height="100" viewBox="0 0 659 100" style="overflow: visible; width: 100%; height: auto;"></svg></div></div></div><div class="dashboard-stat"><div class="deliveries-stat">2</div><div class="deliveries-title">Messages delivered per second</div></div></div>
```

The only aria attributes present are `aria-labelledby` and `aria-describedby`. There is no `aria-label`.

So, I think it could be safely removed. I am not doing it in this PR. Maybe later.